lib/tagexts.c

```diff
@@ -532,7 +532,7 @@ static int filerequireTag(Header h, rpmtd td, headerGetFlags hgflags)
 
 /* I18N look aside diversions */
 
-#if defined(ENABLE_NLS)
+#if defined(ENABLE_NLS) && (defined(__GLIBC__) || !defined(__linux__))
```

Comment:
AFAICS this isn't specific to Linux in any way, just GNU gettext/glibc. In any case, it'd be better to just test for the symbol directly (in cmake).

Reply:
like this?